src/structures/Guild.js

```diff
@@ -304,6 +310,24 @@ class Guild {
   }
 
   /**
+   * AFK voice channel for this guild
+   * @type {?VoiceChannel}
+   * @readonly
+   */
+  get afkChannel() {
+    return this.client.channels.get(this.afkChannelID);
+  }
+
+  /**
+   * System channel for this guild
+   * @type {?GuildChannel}
```

Comment:
That is always a TextChannel (or null), the client only allow selecting those and manually patching it to a voice one removes it.